config/provider.js

```diff
@@ -8,22 +8,26 @@ export const provider = {
     keys: ['some secret key', 'and also the old rotated away some time ago', 'and one more'],
   },
   claims: {
-    family_name: ['family_name'],
-    given_name: ['given_name'],
-    nickname: ['nickname'],
-    gender: ['gender'],
-    preferred_username: ['preferred_username'],
-    address: ['address'],
-    email: ['email'],
-    birth: ['birthdate', 'birthregion', 'birthcountry', 'birthplace'],
+    openid: ['sub'], // Identifiant technique (sub) de l'utilisateur au format OpenIDConnect
```

Comment:
you can remove that entrythe lib not which claims to send back

Reply:
@sandybemonkey can you propose a PR ? I'm eager to see more contribution on this project ?